### PR TITLE
New tolsoft details

### PR DESCRIPTION
**Pull request by muffato**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/sanger-tol/variantcalling/tree/main/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by muffato** [comment]
This shouldn't work because the magic keywords being considered are the nf-core ones
@sanger-tolsoft fix linting

**Comment by muffato** [comment]
But this should work:
@nf-core-bot fix linting

**Comment by muffato** [comment]
This should work because I've (temporarily) allowed the workflow on this branch:
@sanger-tolsoft fix linting

File: .github/workflows/fix-linting.yml [hunk 1]
```diff
@@ -1,28 +1,31 @@
 name: Fix linting from a comment
 on:
+  push:
+    branches:
+      - new_tolsoft_details
```
Comment (gq1) [inline .github/workflows/fix-linting.yml:5]: remove this parts now?

**Review by gq1 — APPROVED** [review]

**Pull request merged**